test/unit/host_test.rb

```diff
@@ -598,13 +598,13 @@ def teardown
 
   context "location or organizations are not enabled" do
     before do
-      SETTINGS[:locations_enabled] = false
-      SETTINGS[:organizations_enabled] = false
+      @original_loc, SETTINGS[:locations_enabled] = SETTINGS[:locations_enabled], false
```

Comment:
+1 for keeping the original values: it makes the tests survive whatever the default is